tests/steps/happy_path.py

```diff
@@ -764,3 +781,59 @@ def step_impl(context, DEVICE_ID):
     for doc in result:
         delete = devices.remove({"id": DEVICE_ID})
         print ("Deleted device {}, result: {}".format(DEVICE_ID, delete))
+
+
+@step("a service and subservice are provisioned")
+def service_subservice_default_provision(context):
+    """
+    :type context: behave.runner.Context
+    """
+    json_data_s = """{
+                                    "DOMAIN_ADMIN_USER": "cloud_admin",
+                                    "NEW_SERVICE_ADMIN_PASSWORD": "4passw0rd",
+                                    "KEYPASS_PORT": "8080",
```

Comment:
If you a re using ORC api then KEYPASS_PORT, KEYSTONE_PORT, KEYSPASS_HOST has no sense